packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.Props.ts

```diff
@@ -61,4 +61,10 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
    * @defaultvalue labelText
    */
   ariaLabel?: string;
-}
+
+  /**
+   * Whether or not the SearchBox is inline.
+   * @default false
+   */
+  inline?: boolean;
```

Comment:
Is this the right prop name? I didn't know what `inline` meant when i read it and assumed it was display: inline.

Reply:
TextField has `underlined`. We should be consistent.

There was a time where we discussed collapsing SearchBox and TextField because they're so similar. We should make sure that the props are very consistent.

Reply:
Consistent prop names is important and this specific change makes sense to me, but we should make sure this is represented in the toolkit somehow. I'm pretty sure this PR started with "inline" because that's how it's named in the toolkit. I don't think necessarily want to rename the toolkit example, but maybe we have some technical indicator for the property used to access it. Thoughts?

Reply:
I believe "inline" is used in the toolkit based on this particular SearchBox's intended use within the CommandBar. This distinction is part of why I was second guessing (see my above comment) even dedicating a whole prop / example to the Underlined/Inline SearchBox. CommandBar has an "isSearchBoxVisible" prop and renders it's own SearchBox it seems - it is not drawing from the SearchBox component.

Here is the toolkit example I am drawing from:
![image](https://user-images.githubusercontent.com/16619843/31096169-212c5c78-a770-11e7-9491-93f3e4ac5a84.png)

Reply:
It seems like it would make sense to rename this as 'Search Box Underline (Command Bar)' in the toolkit, if that makes things more clear. We have a 'Text Box Underline' component in the toolkit so it makes sense to be consistent with the 'Underline' term.